ndbench-api/src/main/java/com/netflix/ndbench/api/plugin/NdBenchAbstractClient.java

```diff
@@ -32,7 +32,7 @@
  *
  * @author vchella, pencal
  */
-public interface NdBenchAbstractClient<W> {
+public interface NdBenchAbstractClient<K, W> {
```

Comment:
Please document the addition of parameter K. Also, is it OK not to narrow down the expected superclass/interfaces for K?